src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -239,6 +245,7 @@ import trackTransactionEvent from '@/util/trackTransactionEvent';
 import checkoutUtils from '@/plugins/checkout-utils-mixin';
 import updateLoanReservationTeam from '@/graphql/mutation/updateLoanReservationTeam.graphql';
 import CampaignHero from '@/components/CorporateCampaign/CampaignHero';
+import DynamicHeroClassic from '@/components/Contentful/DynamicHeroClassic';
```

Comment:
As a potential optimization, you might want to try and use dynamic imports for both hero components as well...

```
ex. const CampaignHero = () => import('@/components/CorporateCampaign/CampaignHero');
```

Reply:
done